fastai/data/external.py

```diff
@@ -80,7 +80,7 @@ class URLs():
     IMDB                = f'{S3_NLP}imdb.tgz'
     IMDB_SAMPLE         = f'{URL}imdb_sample.tgz'
     ML_SAMPLE           = f'{URL}movie_lens_sample.tgz'
-    ML_100k             = 'https://files.grouplens.org/datasets/movielens/ml-100k.zip'
+    ML_100k             = 'http://files.grouplens.org/datasets/movielens/ml-100k.zip'
```

Comment:
I believe this change should be reverted - https is the correct protocol to use AFAIK.

Reply:
Yes, indeed. I did not realize there was a change made there.